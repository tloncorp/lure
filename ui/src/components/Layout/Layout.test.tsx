/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { describe, expect, it } from 'vitest';
import renderer from 'react-test-renderer';
import Layout from './Layout';

describe('Layout', () => {
  it('renders as expected', () => {
    const tree = renderer
      .create(
        <Layout>
          <p>content</p>
        </Layout>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
