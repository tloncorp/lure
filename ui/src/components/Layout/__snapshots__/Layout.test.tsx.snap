// Vitest Snapshot v1

exports[`Layout > renders as expected 1`] = `
<div
  className="layout"
>
  <main
    className="main"
  >
    <p>
      content
    </p>
  </main>
</div>
`;
